src/components/Landing/Landing.tsx

```diff
@@ -22,41 +23,65 @@ const LandingPage: React.FC<{ collections?: Opds2Feed }> = ({
   collections,
 }) => {
   const subHeader = (
-    <span>
-      Find millions of digital books for research from multiple sources
-      world-wide--all free to read, download, and keep. No library card
-      required. This is an early beta test, so we want your feedback!{" "}
-      <Link href="/about" _hover={{ color: "ui.white" }} color="ui.white">
-        Read more about the project
-      </Link>
-      .
-    </span>
+    <Box>
+      <span>
+        Find millions of digital books for research from multiple sources
+        world-wide--all free to read, download, and keep. No library card
+        required. This is an early beta test, so we want your feedback!{" "}
+        <Link href="/about" color="ui.link.primary">
+          Read more about the project
+        </Link>
+        .
+      </span>
+      <Box marginTop="s">
+        Search the World's Research Collections
+        <SearchForm isFullWidth />
```

Comment:
is there a way we can add the appropriate style without needing to pass down a prop? maybe by wrapping the `<SearchForm /> ` in a `<Box />` and adding additional style properties there